### PR TITLE
ER-641 code clean up

### PR DESCRIPTION
**Pull request by leewadhams**
See ticket for details.

### CODE REVIEW

File: src/Shared/Recruit.Vacancies.Client/Infrastructure/Services/IEmployerDashboardService.cs [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 namespace Esfa.Recruit.Vacancies.Client.Infrastructure.Services
 {
-    public interface IDashboardService
+    public interface IEmployerDashboardService
```
Comment (sachpatel) [inline src/Shared/Recruit.Vacancies.Client/Infrastructure/Services/IEmployerDashboardService.cs]: Could this be left as `IDashboardService` as the other dashboards could also implement this interface and the method names are general.

File: src/Shared/Recruit.Vacancies.Client/Infrastructure/Events/QueueNames.cs [hunk 2]
```diff
@@ -4,7 +4,7 @@ public static class QueueNames
     {
         public const string VacancyEventsQueueName = "vacancy-events-queue";
         public const string VacancyReviewEventsQueueName = "vacancy-review-events-queue";
-        public const string DashboardQueueName = "dashboard-queue";
+        public const string EmployerDashboardQueueName = "employer-dashboard-queue";
```
Comment (sachpatel) [inline src/Shared/Recruit.Vacancies.Client/Infrastructure/Events/QueueNames.cs]: Would you mind prefixing this queue name with `generate-`. This was my fault for giving it a bad name in the first place.

File: src/src.sln [hunk 3]
```diff
@@ -0,0 +1,6 @@
+
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio 2013
```
Comment (sachpatel) [inline src/src.sln]: 🤣

File: src/Shared/Recruit.Vacancies.Client/Domain/Events/ApplicationSubmittedEvent.cs [hunk 4]
```diff
@@ -0,0 +1,11 @@
+﻿using Esfa.Recruit.Vacancies.Client.Domain.Messaging;
+using MediatR;
+
+namespace Esfa.Recruit.Vacancies.Client.Domain.Events
+{
+    // Note: Doesn't implement IApplicationReviewEvent as it's an externally published event.
```
Comment (sachpatel) [inline src/Shared/Recruit.Vacancies.Client/Domain/Events/ApplicationSubmittedEvent.cs]: Should we maybe put this declaration closer to the one job that uses it as it seems very specific maybe.

File: src/Employer/Employer.Web/Mappings/Extensions/IApprenticeshipProgrammeMapperExtensions.cs [hunk 5]
```diff
@@ -3,9 +3,9 @@
 using Esfa.Recruit.Employer.Web.ViewModels.Part1.Training;
 using Esfa.Recruit.Vacancies.Client.Domain.Entities;
 
-namespace Esfa.Recruit.Employer.Web.Extensions
+namespace Esfa.Recruit.Employer.Web.Mappings.Extensions
 {
-    public static class IApprenticeshipProgrammeExensions
+    public static class IApprenticeshipProgrammeMapperExtensions
```
Comment (sachpatel) [inline src/Employer/Employer.Web/Mappings/Extensions/IApprenticeshipProgrammeMapperExtensions.cs]: Does ApprenticeshipProgrammes type need an interface?It has resulted in this class sounding interfacey 🦊

File: src/Shared/Recruit.Vacancies.Client/Infrastructure/Services/IEmployerDashboardService.cs (hunk 1 above)
Comment (leewadhams) [reply src/Shared/Recruit.Vacancies.Client/Infrastructure/Services/IEmployerDashboardService.cs]: 👍

File: src/Shared/Recruit.Vacancies.Client/Infrastructure/Events/QueueNames.cs (hunk 2 above)
Comment (leewadhams) [reply src/Shared/Recruit.Vacancies.Client/Infrastructure/Events/QueueNames.cs]: Done

File: src/src.sln (hunk 3 above)
Comment (leewadhams) [reply src/src.sln]: Removed. That was interestingly created by either Visual Studio Code or Rider.

File: src/Shared/Recruit.Vacancies.Client/Domain/Events/ApplicationSubmittedEvent.cs (hunk 4 above)
Comment (leewadhams) [reply src/Shared/Recruit.Vacancies.Client/Domain/Events/ApplicationSubmittedEvent.cs]: But it is still a domain event isn't it? We're just acting on it as opposed to raising it.

Comment (sachpatel) [reply src/Shared/Recruit.Vacancies.Client/Domain/Events/ApplicationSubmittedEvent.cs]: 👍 I was just thinking because it is used in one place.

**Pull request merged**